src/sentry/integrations/vercel/uninstall.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+
+import logging
+
+from django.views.decorators.csrf import csrf_exempt
+from sentry.api.base import Endpoint
+from sentry.models import AuditLogEntryEvent, Integration, OrganizationIntegration, Organization
+from sentry.utils.audit import create_audit_entry
+from sentry.web.decorators import transaction_start
+
+logger = logging.getLogger("sentry.integrations.vercel.uninstall")
+
+
+class VercelUninstallEndpoint(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    @csrf_exempt
+    def dispatch(self, request, *args, **kwargs):
+        return super(VercelUninstallEndpoint, self).dispatch(request, *args, **kwargs)
+
+    @transaction_start("VercelUninstallEndpoint")
+    def post(self, request):
```

Comment:
@MeredithAnya This isn't a `DELETE` method? That's what the [docs](https://vercel.com/docs/integrations#delete-hooks) seem to imply.

Reply:
ahh yes, will update ! ty